packages/mobile/src/exchange/ExchangeReview.tsx

```diff
@@ -66,13 +70,32 @@ export class ExchangeReview extends React.Component<Props, State> {
   }
 
   onPressConfirm = () => {
-    const makerToken = this.state.makerToken
+    const { makerToken, inputToken, inputAmount } = this.state
     const makerAmount = this.getMakerAmount()
+    // BEGIN: Analytics
+    const isDollarToGold = inputToken === CURRENCY_ENUM.DOLLAR
+    const cGLDToCUSDExchangeRate = getRateForMakerToken(
+      this.props.exchangeRatePair,
+      this.state.makerToken,
+      CURRENCY_ENUM.DOLLAR
+    )
+    const goldAmount = isDollarToGold ? makerAmount : inputAmount
+    const dollarsAmount = isDollarToGold ? inputAmount : makerAmount
+    const localCurrencyAmount = convertDollarsToLocalAmount(
+      dollarsAmount,
+      this.props.localCurrencyExchangeRate
+    )
+    CeloAnalytics.track(
+      isDollarToGold ? CustomEventNames.gold_buy_confirm : CustomEventNames.gold_sell_confirm,
+      {
+        localCurrencyAmount,
```

Comment:
Are these on the properties whitelist? I think they don't get included if they're not. I didn't set up analytics so I'm not 100% sure how it works but I think props need to be whitelisted, which confused many people tbh https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/analytics/constants.ts#L207

Reply:
I've updated `PROPERTY_PATH_WHITELIST` - looks a bit odd to me, so I guess we should work really soon on this `TODO` right above it :)